chainer/link.py

```diff
@@ -1202,6 +1226,15 @@ def namedparams(self, include_uninit=True):
             for path, param in link.namedparams(include_uninit):
                 yield prefix + path, param
 
+    def namedpersistent(self):
+        # type: () -> tp.Iterator[tp.Tuple[str, tp.NdArray]]
+        for ret in super(ChainList, self).namedpersistent():
+            yield ret
+        for idx, link in enumerate(self._children):
+            prefix = '/{}'.format(idx)
```

Comment:
```suggestion
            prefix = '/%d' % idx
```
to be consistent with `namedparams` and `namedlinks`.